### PR TITLE
Handle "Validation Check Failed" of Nv repo in AY installation

### PR DESCRIPTION
**Pull request by mimi1vx**
- Related ticket: https://progress.opensuse.org/issues/55220
- Verification run: http://quasar.suse.cz/tests/3562#step/installation/5

### CODE REVIEW

File: tests/autoyast/installation.pm [hunk 1]
```diff
@@ -239,6 +239,11 @@ sub run {
         elsif (match_has_tag('autoyast-error')) {
             die 'Error detected during first stage of the installation';
         }
+        elsif (match_has_tag('nvidia-validation-failed')) {
+           # nvidia repositories are unstable and really not needed for anything
```
Comment (czerw) [inline tests/autoyast/installation.pm]: I think record info with short comment could be here to make it more visible.

Comment (mimi1vx) [reply tests/autoyast/installation.pm]: why ? is unsupported thirdparty problem ...

Comment (baierjan) [reply tests/autoyast/installation.pm]: Or soft-fail with bsc#1144831

Comment (ostruzina) [reply tests/autoyast/installation.pm]: I went for soft failure needle with this reference

Comment (czerw) [reply tests/autoyast/installation.pm]: > why ? is unsupported thirdparty problem ..., maybe if we want some 'statistics'

At least to show that something non standard, but not critical happened.

**Pull request merged**